src/components/board.js

```diff
@@ -1,22 +1,24 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
-export const Board = props => {
-    return (
-        <div className="root">
-            <div className="grid-container">{props.lights}</div>
-            <p>{props.numMoves}</p>
-            <button onClick={props.reset}>Reset</button>
-            {props.hasWon && (
-                <div>Congratulations, you Won! Click reset to play again.</div>
-            )}
-        </div>
-    );
-};
+export const WIN_TEXT = 'Congratulations, you Won! Click reset to play again.';
+
+const Board = props => (
+    <div className="root">
+        <div className="grid-container">{props.lights}</div>
+        <p>{props.numMoves}</p>
+        <button data-testid="reset" onClick={props.reset}>
+            Reset
+        </button>
+        {props.hasWon && <div data-testid="win-div">{WIN_TEXT}</div>}
```

Comment:
Looks great - a note specifically about **React Native**:

This is a bit of foreshadowing... this pattern works great in React (web), but has a tendency to cause problems in React Native (specifically referring to `[expression] && [expression]` which of course we use to coerce some value into a boolean in order to conditionally evaluate and return the following expression). If the first expression is falsey, then the whole combined expression evaluates to `false` which React Native chokes on (can't render text outside of a `<Text />` component) - again totally fine in React web. For RN, instead you use a ternary and explicitly return `null`: 

```jsx
{props.hasWon ? <whatever /> : null}
```